tests/unit/general/test_arrays_by_name.cpp

```diff
@@ -0,0 +1,129 @@
+// Copyright (c) 2010-2023, Lawrence Livermore National Security, LLC. Produced
+// at the Lawrence Livermore National Laboratory. All Rights reserved. See files
+// LICENSE and NOTICE for details. LLNL-CODE-806117.
+//
+// This file is part of the MFEM library. For more information and source code
+// availability visit https://mfem.org.
+//
+// MFEM is free software; you can redistribute it and/or modify it under the
+// terms of the BSD-3 license. We welcome feedback and contributions, see file
+// CONTRIBUTING.md for details.
+
+#include "mfem.hpp"
+#include "unit_tests.hpp"
+
+using namespace mfem;
+
+void FillArraysByName(ArraysByName<int> &abn)
```

Comment:
Perhaps include duplicates in the arrays, and verify that duplicates are removed?

Reply:
...and the new tests fail...

Reply:
I guess I'm still learning how to use c++-11